modules/ROOT/pages/migration/index.adoc

```diff
@@ -44,32 +71,1036 @@ a|
+This is consistent with how queries work with the same directive configuration.
+Previously, mutations would always follow the explicit `direction` value when matching existing relationships, which could lead to inconsistent behavior between queries and mutations.
+
+=== Moved `where` field in nested update operations
+
+The `where` field for nested update operations has been removed in favor of the `where` inside the nested update input.
+
+[cols="1,1"]
+|===
+| Before | After
+
+a|
+[source, graphql, indent=0]
+----
+mutation {
+  updateUsers(
+    where: { name: { eq: "Darrell" } }
+    update: {
+      posts: {
+        where: { node: { title: { eq: "Version 7 Release Notes" } } }
+        update: { node: { title: { set: "Version 7 Release Announcement" } } }
+      }
+    }
+  ) {
+    users {
+      name
+      posts {
+        title
+      }
+    }
+  }
+}
+----
+a|
+[source, graphql, indent=0]
+----
+mutation {
+  updateUsers(
+    where: { name: { eq: "Darrell" } }
+    update: {
+      posts: {
+        update: {
+          where: { node: { title: { eq: "Version 7 Release Notes" } } }
+          node: { title: { set: "Version 7 Release Announcement" } }
+        }
+      }
+    }
+  ) {
+    users {
+      name
+      posts {
+        title
+      }
+    }
+  }
+}
+----
+|===
+
+=== Changed behavior for `single` and `some` filters on relationships to unions
+
+The behavior of `single` and `some` filters when used with relationships to union types has been fixed, which represents a breaking change from the previous incorrect implementation.
+
+Consider this schema with a union type and a relationship to it:
+
+[source, graphql, indent=0]
+----
+union Production = Movie | Series
+
+type Actor @node {
+    name: String!
+    actedIn: [Production!]! @relationship(type: "ACTED_IN", direction: OUT)
+}
+----
+
+Previously, when using `single` or `some` filters with unions, conditions inside these filters were evaluated separately for each concrete type in the union, requiring all of them to match.
+This behavior was incorrect.
+
+[source, graphql, indent=0]
+----
+query {
+    actors(
+        where: {
+            actedIn: { single: { Movie: { title: { contains: "The Office" } }, Series: { title: { endsWith: "Office" } } } }
+        }
+    ) {
+        name
+    }
+}
+----
+
+The fixed behavior in version 7.0.0:
+
+* `single`: Now correctly returns actors with exactly one related node across the whole union, rather than per type.
+* `some`: Now correctly returns actors with at least one matching related node of any type in the union.
+
+This change provides more logical and consistent results, but may affect existing queries that relied on the previous behavior.
+This fix applies to both the new filter syntax and the deprecated filters (e.g., `actedIn_SINGLE` and `actedIn_SOME`).
+
+=== List of nullable elements no longer supported
```

Comment:
@rsill-neo4j  Can you validate the following points that we  have just have merged?